xprocspec-runner/pom.xml

```diff
@@ -106,6 +106,11 @@
         <artifactId>maven-bundle-plugin</artifactId>
         <configuration>
           <instructions>
+            <!-- must match version from framework-bom -->
+            <Import-Package>
+              net.sf.saxon.*;version="[9.5.1,9.5.2)",
```

Comment:
why do we need this version range here, and not 9.8?

Reply:
This happens on several places: it looks as if some modules are configured to work with the 9.5 API, and some with the 9.8 API?

Reply:
Probably an oversight because it looks like the framework-bom has 9.8.